Test/allocated1/dafny0/CompilationErrors.dfy.expect

```diff
@@ -5,6 +5,7 @@ CompilationErrors.dfy(83,4): Warning: note, this loop has no body (loop frame: x
 CompilationErrors.dfy(85,4): Warning: note, this loop has no body (loop frame: i, x)
 
 Dafny program verifier finished with 5 verified, 0 errors
+CompilationErrors.dfy(3,8): Error: the included file CompilationErrors.dfy contains error(s)
```

Comment:
That message is weird. How come it references itself?

Reply:
There are two files with the same name, one includes the other.